packages/dds/tree/src/test/shared-tree/fuzz/composeVsIndividual.fuzz.spec.ts

```diff
@@ -148,24 +170,42 @@ describe("Fuzz - composed vs individual changes", () => {
 			DDSFuzzTestState<SharedTreeTestFactory>
 		> = {
 			workloadName: "SharedTree",
-			factory: new SharedTreeTestFactory(() => {}),
+			factory: new SharedTreeTestFactory(createOnCreate(undefined)),
 			generatorFactory,
 			reducer: fuzzComposedVsIndividualReducer,
 			validateConsistency: () => {},
 		};
 		const emitter = new TypedEventEmitter<DDSFuzzHarnessEvents>();
 		emitter.on("testStart", (initialState: BranchedTreeFuzzTestState) => {
 			initialState.main = viewFromState(initialState, initialState.clients[0]);
-			initialState.branch = initialState.main.fork() as FuzzTransactionView;
-			initialState.branch.currentSchema = initialState.main.currentSchema;
+
+			const branchCheckout = initialState.main.checkout.fork();
+			const treeSchema = simpleSchemaFromStoredSchema(
+				initialState.main.checkout.storedSchema as TreeStoredSchemaRepository,
+			);
+			const branchView = viewCheckout(
+				branchCheckout,
+				new TreeViewConfiguration({ schema: treeSchema }),
+			) as FuzzTransactionView;
+
+			const nodeSchema = nodeSchemaFromTreeSchema(treeSchema);
+
+			branchView.currentSchema =
+				nodeSchema ?? assert.fail("nodeSchema should not be undefined");
+			initialState.branch = branchView;
 			initialState.branch.checkout.transaction.start();
```

Comment:
we leave a transaction open on test start?

Reply:
The `composeVsIndividual` tests start the transaction on the fork at the beginning of the test, and commit them at the end of the test (with the transaction weights set to 0), so that it composes the edits. After committing the transaction, we merge the fork into the main, and check that we end up with the same state.